### PR TITLE
Added function to create flow with resampled base

### PR DESCRIPTION
**Pull request by VincentStimper**
* Added function to create normalizing flow model with resampled base distribution

### CODE REVIEW

File: experiments/make_flow/make_realnvp_normflow.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import normflows as nf
+import larsflow as lf
```
Comment (lollcat) [inline experiments/make_flow/make_realnvp_normflow.py:3]: Should this be added to the requirements?

**Comment by lollcat** [comment]
To clarify: For training I just call `flow.reverse_kld` and for log importance weights `target.log_prob(x) - flow.log_prob(x)`?

**Review by lollcat — APPROVED** [review]

File: experiments/make_flow/make_realnvp_normflow.py (hunk 1 above)
Comment (VincentStimper) [reply experiments/make_flow/make_realnvp_normflow.py:3]: Done.

**Comment by VincentStimper** [comment]
> To clarify: For training I just call `flow.reverse_kld` and for log importance weights `target.log_prob(x) - flow.log_prob(x)`?

Yes, but you can equally well use https://github.com/lollcat/fab-torch/blob/9384275f042918b1d3bb808167e329fb96d3410b/fab/core.py#L82 instead as you probably have done before.

**Pull request merged**